redis-oplog.js

```diff
@@ -36,4 +36,6 @@ export {
 
 if (Meteor.settings.redisOplog) {
     init(Meteor.settings.redisOplog);
+} else {
```

Comment:
make sure to do the check if (Meteor.isDevelopment)

Reply:
Why not have the error message in both?

Reply:
There is a way to initiate REdisOplog without settings. Though not recommended.
`RedisOplog.init(settings)`